sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -5,6 +5,10 @@ select format_string();
 -- A pipe operator for string concatenation
 select 'a' || 'b' || 'c';
 
+-- split function
+select split('aa1cc2ee', '[1-9]+', 2);
+select split('aa1cc2ee', '[1-9]+');
+
```

Comment:
Can you move these tests to the end of this file in order to decrease unnecessary changes in the golden file.